sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,906 @@
+a fixed amount of time on the happy path** if both Stackers and miners operate with a BFT honest
+majority.  If they do not, then peg-outs of arbitrary amounts of BTC 
+are fulfilled eventually by _redirecting_ Stackers' PoX payouts to fund peg-out
+requests.
+
+To support this, sBTC offers two modes of operation:
+the Normal mode, and the Recovery mode.  In **Normal mode**, the sBTC asset is
+backed 1:1 by BTC sent to a wallet controlled by a large fraction of Stackers,
+as measured by the fraction of locked STX they represent.  Each time BTC is
+sent to this wallet (a peg-in operation), an equal number of sBTC are
+automatically minted to an address of the sender's choosing.  Stackers respond
+to peg-out requests by sending BTC to the requester's Bitcoin address, thereby
+maintaining the peg.  The act of responding to peg-out requests automatically
+destroys the requester's equivalent amount of sBTC on the Stacks chain.
+
+If the Normal mode encounters a liveness failure for any reason (including loss
+of BTC), the system transitions to a Recovery mode until enough Stackers come
+back online.  In **Recovery mode**, a fraction of the PoX payouts are
+redirected to peg-out requests such that eventually, all outstanding requests
+will be fulfilled, even if the Stackers _never_ come back online.  While
+considerably slower than Normal mode, the design of Recovery mode ensures all
+sBTC can be redeemed so long as the Stacks blockchain and PoX are online.
+
+This proposal, if ratified, would be executed by at least two hard forks:  one
+to implement Normal mode, and a follow-on hard fork to implement Recovery mode.
+This SIP is designed to leverage features proposed by SIP-021, and can only
+activate if SIP-021 is ratified beforehand or concurrently with this SIP.
+
+This proposal describes Normal mode.  Recovery mode will be fully described in a
+future SIP, and will have a separate activation procedure.  The Recovery mode
+SIP cannot activate without this SIP first being ratified.
+
+# Specification
+
+The proposed SIP-021 upgrade would do away with Stacks forks, which vastly
+simplifies the sBTC design.  Importantly, miners and Stackers cannot "roll back"
+sBTC operations, meaning that peg-in and peg-out operations follow at-most-once
+semantics.  This lack of "fork risk" means that the system can support
+arbitrarily large amounts of BTC pegged-in and pegged-out, assuming that at
+least 70% of Stackers by weight are trustworthy.
+
+The sBTC system state is materialized in a boot contract called `.sbtc`.  This
+contract, like `.pox-4`, is managed in part by the node itself.  In particular,
+the node writes sBTC state from Bitcoin into the data space of `.sbtc` (such as
+via a private function that only the node can call), so that it can be accessed
+by smart contracts in Stacks.
+
+The following sections describe the Normal mode of operation.  Recovery mode is
+left for future work, but is described in a separate section after Normal mode.
```

Comment:
earlier in the doc you say recovery mode will be described in a future SIP

Reply:
A synopsis is provided.  I'll make that clear here.